README.md

```diff
@@ -1,2 +1,98 @@
-# community
-Cloud Hypervisor Community
+- [About Cloud Hypervisor](#about-cloud-hypervisor)
+- [Governance](#governance)
+  * [Projects](#projects)
+  * [Developers](#developers)
+    + [Contributors](#contributors)
+    + [Maintainers](#maintainers)
+    + [Becoming a project maintainer](#becoming-a-project-maintainer)
+  * [Architecture Committee](#architecture-committee)
+    + [Elections](#elections)
+
+# About Cloud Hypervisor
+
+Cloud Hypervisor is an open source organization and community hosting several
+projects aiming at building a modern and innovative cloud virtualization
+ecosystem.
+
+# Governance
+
+The Cloud Hypervisor organization is governed according to the
+["four opens"](https://governance.openstack.org/tc/reference/opens.html), which
+are open source, open design, open development, and open community.
+
+Technical decisions are made by technical contributors and a representative
+Architecture Committee.
+
+## Projects
+
+The Cloud Hypervisor organization is composed of several projects, like e.g. the
+[cloud-hypervisor](https://github.com/cloud-hypervisor/cloud-hypervisor) or the
+[rust-hypervisor-firmware](https://github.com/cloud-hypervisor/rust-hypervisor-firmware)
+ones.
+
+A project is the primary unit of collaboration, therefore each project has its
+own repository and maintainers team. All projects follow the
+[Code of Conduct](CODE_OF_CONDUCT.md).
+
+## Developers
+
+There are two roles relevant to the Cloud Hypervisor projects:
+
+### Contributors
+
+Anyone that contributed to one of the Cloud Hypervisor projects within the
+last 12 months is a *Contributor*. Any merged Pull Request is considered a valid
+contribution.
+
+Contributions are not limited to code alone. Documentation, tests, tools, project
+artifacts are all valuable ways to contribute to a Cloud Hypervisor project.
+
+Project contributions will be reviewed by the project maintainers and should
+pass all applicable tests.
+
+### Maintainers
+
+Each project has one or more *Maintainers*. Project maintainers are first and
+foremost and active *Contributor* to the project and are responsible for:
```

Comment:
I'm not sure if my English is failing me or there's something wrong with the second line. Maybe it should be "an active" instead of "and active"?

Reply:
Fixed now, thanks.